### PR TITLE
Refactor controller

### PR DESCRIPTION
**Pull request by jakeheft**
Description:  
* Refactored all controller endpoints to hit service, added tests for all endpoints, added home services, fixed serializer so that all tests pass in TravisCI

Fixes #

* [X] All tests passing
* [ ] Lint code
* [X] Checked local host
* [ ] Updated README

### CODE REVIEW

File: config/environment.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 require "bundler"
 Bundler.require(:default, ENV['SINATRA_ENV'])
 Bundler.require(Sinatra::Base.environment)
-require "active_support/deprecation"
-require "active_support/all"
+# require "active_support/deprecation"
+# require "active_support/all"
```
Comment (Stearnzy) [inline config/environment.rb]: Will you be deleting these comments?

File: spec/application_controller_spec.rb [hunk 2]
```diff
@@ -107,7 +215,7 @@ def app
       expect(biz).to have_key(:name)
       expect(biz[:name]).to be_an(String)
       expect(biz).to have_key(:is_closed)
-      expect(biz[:is_closed]).to be_in([true, false])
+      expect(biz[:is_closed]).to be(true).or be(false)
```
Comment (Stearnzy) [inline spec/application_controller_spec.rb:218]: Cool I didn't know you could do this!

**Review by Stearnzy — APPROVED** [review]

**Review by Oxalisviolacea — APPROVED** [review]

**Comment by Oxalisviolacea** [comment]
Great refactor! We should probably lint the code too.

**Review by BJSherman80 — APPROVED** [review]

**Comment by jakeheft** [comment]
This was merged, but Brett made another PR that this code will be on and we'll lint the whole thing

**Pull request merged**